JenifarFathimaH/00 - Projects/04 - Frontend Projects/E-Commerce Project/Assets/JS/Admin/AddProduct.js

```diff
@@ -0,0 +1,99 @@
+const PRODUCT_NAME = document.getElementById("inp-productName");
+const PRODUCT_ID = document.getElementById("inp-productId");
+const PRODUCT_PRICE = document.getElementById("inp-productPrice");
+const PRODUCT_DESCRIPTION = document.getElementById("inp-productDescription");
+const ALERT_MESSEGE = document.querySelector('.alert_messege');
+
+const SUBMIT_BUTTON = document.getElementById("inp-SubmitButton");
+const UPDATE_BUTTON = document.getElementById("inp-UpdateButton");
+
+const EDIT_INDEX = parseInt(localStorage.getItem("editIndex"));
+
+const PRODUCT_DETAILS = JSON.parse(localStorage.getItem("productDetails")) || [];
+
+let AddProductMethods = {};
+let EditAndUpdateMethods = {};
+
+AddProductMethods.FillProductDetails = function () {
+    let strProductName = PRODUCT_NAME.value;
+    let nProductId = PRODUCT_ID.value;
+    let nProductPrice = PRODUCT_PRICE.value;
+    let strProductDescription = PRODUCT_DESCRIPTION.value;
+
+    if (!strProductName || !nProductId || !nProductPrice || !strProductDescription) {
+        ALERT_MESSEGE.innerHTML = `<p>All fields are Required<p>`;
```

Comment:
Create a separete object for messages. Discuss with Akshaya.,

Also, do not include <p>(elements) inside the message

E.g.,
<div><p id="error"></p></div>

document.getElementById("error").innerHTML = "All fields required...";

Reply:
Done